src/test/java/com/google/api/codegen/testsrc/libraryproto/library_gapic.yaml

```diff
@@ -246,6 +246,11 @@ interfaces:
         - parameter: edition
           sample_argument_name: edition
       on_success:
+      - comment:
+        - "%% %% %% output handling %% %% %% %%"
+      - comment:
+        - "Do something with %s one by one"
+        - book_names
```

Comment:
I was struggling to figure out why this was separated out into `%s` but then I read the baseline's and saw how it helps with consistent casing/etc!

AWESOME.

Reply:
Guessing this will work within a loop?

I'm guessing we process the object exactly the same, but it might be worth adding a comment into a loop block to verify!

Reply:
Can `book_names` be any string?

– or does it refer to messages/fields/variable tables in any way?

If any string, does this always presume the string_will_be_in_snake_case? (which I like, just clarifying)

Reply:
`book_names` can be any string, and has to be in_snake_case :)

Reply:
Good idea about capturing the in-loop comment in a test!